CorsixTH/Lua/dialogs/resizables/options.lua

```diff
@@ -183,6 +201,10 @@ function UIOptions:UIOptions(ui, mode)
   self:addBevelPanel(165, customise_y_pos, BTN_WIDTH, 30, col_bg):setLabel(_S.options_window.folder)
     :makeButton(0, 0, BTN_WIDTH, 30, nil, self.buttonFolder):setTooltip(_S.tooltip.options_window.folder_button)
 
+  -- "Hotkeys" button
+  local hotkey_y_pos = self:_getOptionYPos() + 10
```

Comment:
Does it look better centered?

Reply:
How about this?
![temp_01](https://user-images.githubusercontent.com/20325471/50542296-be1f8680-0b7f-11e9-8f0f-3fdb5066ec67.png)